eng/Build.ps1

```diff
@@ -563,12 +563,19 @@ try {
 
     $script:BuildMessage = "Failure building product"
     if ($restore -or $build -or $rebuild -or $pack -or $sign -or $publish -and -not $skipBuild -and -not $sourceBuild) {
+        $originalSignValue = $sign
+        if ($msbuildEngine -eq "dotnet") {
+            # Building FSharp.sln and VisualFSharp.sln with .NET Core MSBuild
+            # don't produce any artifacts to sign. Skip signing in this case.
+            $sign = $False
+        }
         if ($noVisualStudio) {
             BuildSolution "FSharp.sln" $False
         }
         else {
             BuildSolution "VisualFSharp.sln" $False
         }
+        $sign = $originalSignValue
```

Comment:
Does the oriignal value need to be restored?

Reply:
Yeah. It's for the case that the remaining solutions get built as well, eg `FSharp.Benchmarks.sln` and `Microsoft.FSharp.Compiler.sln`